src/pages/tickets/ticket-list-page.js

```diff
@@ -1023,19 +1020,24 @@ class TicketListPage extends React.Component {
             )}
             {enabledFilters.includes("ticketTypesFilter") && (
               <div className="col-md-6">
-                <Select
+                <TicketTypesInput
                   placeholder={T.translate(
                     "ticket_list.placeholders.ticket_types"
                   )}
                   name="ticketTypesFilter"
                   value={ticketFilters.ticketTypesFilter}
-                  onChange={(val) =>
-                    this.handleFilterChange("ticketTypesFilter", val)
+                  onChange={(ev) =>
```

Comment:
lets create a helper method to translate the value retrieved by TicketTypesInput to the ones expected by the actions ( rollback action changes )
also review ticket action https://github.com/fntechgit/summit-admin/pull/540/files#diff-e34888831de52961fbe9061efcda081680b4ff79c8211c5b93069c37b2879c2eR199